.prettierrc

```diff
@@ -0,0 +1,3 @@
+{
+  "tabWidth": 2
+}
```

Comment:
Well, technically the whole point of prettier is to not slow anyone down. You can type single quotes and prettier will change it for you (need to add it into your IDE first though, or we can add a pre-commit hook). Mostly I'd leave it though for consistency across our other dev environments, otherwise, I agree with the preference.

Reply:
I'm going to leave as is for consistency but we can revisit as we develop a more formalized open source strategy in the coming weeks/months.